### PR TITLE
login page

### PR DESCRIPTION
**Pull request by MajdTamimi**

### CODE REVIEW

File: src/Component/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
+const footer = () => {
+  
+  return (
+    <footer className="text-[#1c1e21] pl-10 lg:text-start pt-8 lg:pt-7">
+      <div className="max-w-[980px] mx-auto w-[1513px] h-[157px]">
+        <div className="text-[12px] font-inherit w-[684px] lg:w-[980px] h-[28px]">
+          <a href="#" className="text-[#8A8d91] font-custom text-[12px] font-inherit hover:underline mr-2">English (US)</a>
```
Comment (abdalazeezS) [inline src/Component/Footer.jsx]: Don't repeat the code, put them in array and map over it

File: src/Component/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+          <a href="#" className="text-[#8A8d91] font-custom text-[12px] font-inherit hover:underline mr-2">Bahasa Indonesia</a>
+          <a href="#" className="text-[#8A8d91] font-custom text-[12px] font-inherit hover:underline mr-2">Español</a>
+          <a href="#" className="text-[#8A8d91] font-custom text-[12px] font-inherit hover:underline mr-2">Português (Brasil)</a>
+          <a href="#" className="text-[#8A8d91] font-custom text-[12px] font-inherit hover:underline mr-2">Italiano</a>
+          <a href="#" className="text-[#8A8d91] font-custom text-[12px] font-inherit hover:underline">हिन्दी</a>
+          <a role="button" className="inline-block cursor-pointer list-none border border-[#ccd0d5] rounded-[2px] box-border mx-2 font-bold antialiased justify-center px-[8px] text-center text-[#8a8d91] bg-[#f5f6f7] duration-200 hover:bg-[#ebecec]" rel="dialog" href="#" title="Show more languages">
+            <i className="inline-block cursor-pointer antialiased font-bold text-center leading-[18px] text-[#8a8d91] whitespace-nowrap relative bottom-[1px] align-middle"
+              style={{
```
Comment (abdalazeezS) [inline src/Component/Footer.jsx]: إذا بشوف inline css كمان مرة بعمل decline لل pull request

File: src/Component/Footer.jsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+                backgroundPosition: "-66px -35px",
+                width: "12px",
+                height: "12px",
+                textShadow: "none"
+              }}></i>
+          </a>
+        </div>
+        <hr className="pb-2 lg:mt-1 mt-4" />
+        <div className="text-[12px] font-inherit w-[650px] lg:w-[950px] h-[58px] flex flex-wrap">
+          <a href="#" className="text-[#8A8d91] font-custom text-[12px] font-inherit hover:underline mr-4">Sign Up</a>
```
Comment (abdalazeezS) [inline src/Component/Footer.jsx]: same as أختها

File: src/Component/Footer.jsx [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+          <a href="#" className="text-[#8A8d91] font-custom text-[12px] font-inherit hover:underline mr-4">Grups</a>
+          <a href="#" className="text-[#8A8d91] font-custom text-[12px] font-inherit hover:underline mr-4">About</a>
+          <a href="#" className="text-[#8A8d91] font-custom text-[12px] font-inherit hover:underline mr-4">Create ad</a>
+          <a href="#" className="text-[#8A8d91] font-custom text-[12px] font-inherit hover:underline mr-4">Create Page</a>
+          <a href="#" className="text-[#8A8d91] font-custom text-[12px] font-inherit hover:underline mr-4">Developers</a>
+          <a href="#" className="text-[#8A8d91] font-custom text-[12px] font-inherit hover:underline mr-4">Careers</a>
+          <a href="#" className="text-[#8A8d91] font-custom text-[12px] font-inherit hover:underline mr-4">Cookies</a>
+          <a href="#" className="text-[#8A8d91] font-custom text-[12px] font-inherit hover:underline">Ad choices</a>
+          <a href="#" className="inline-block text-[12px] leading-[1.6] cursor-pointer whitespace-nowrap text-[#8a8d91] relative top-[1px] align-middle mr-4"
+            style={{
```
Comment (abdalazeezS) [inline src/Component/Footer.jsx]: same as أختها

**Review by abdalazeezS — CHANGES_REQUESTED** [review]
Resolve my comment please, بدون بلوك

File: src/Component/Footer.jsx (hunk 2 above)
Comment (MajdTamimi) [reply src/Component/Footer.jsx]: 🥲🥲

**Pull request merged**